src/main/java/org/dataloader/registries/ScheduledDataLoaderRegistry.java

```diff
@@ -57,6 +61,86 @@ public Duration getScheduleDuration() {
         return schedule;
     }
 
+    /**
+     * This will combine all the current data loaders in this registry and all the data loaders from the specified registry
+     * and return a new combined registry
+     *
+     * @param registry the registry to combine into this registry
+     *
+     * @return a new combined registry
+     */
+    public ScheduledDataLoaderRegistry combine(DataLoaderRegistry registry) {
+        Builder combinedBuilder = ScheduledDataLoaderRegistry.newScheduledRegistry()
+                .dispatchPredicate(this.dispatchPredicate);
+        combinedBuilder.registerAll(this);
+        combinedBuilder.registerAll(registry);
```

Comment:
Maybe worth checking here, or inside `registerAll` if the other registry is also a `ScheduledDataLoaderRegistry` and if so picking up its dispatchPredicates too.

Reply:
checked - it does